src/rustllvm/PassWrapper.cpp

```diff
@@ -131,12 +131,19 @@ LLVMRustAddPass(LLVMPassManagerRef PM, LLVMPassRef rust_pass) {
 #define SUBTARGET_PPC
 #endif
 
+#ifdef LLVM_COMPONENT_SYSTEMZ
```

Comment:
> link to related libraries when building rust against system llvm

this fixes the travis builds. We could add a build slave that tests building rust against system llvm to avoid these in the future